src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import classNames from 'classnames';
```

Comment:
you can import `classNames` as `cn` and use short version)

Reply:
I'm sorry, but this is a perversion, how I can understand that "cn" it's library for class names?